src/manifest.chrome.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Cyber Citizenship Coefficient",
+  "description": "Build an Chrome Plugin Extension",
```

Comment:
Do we need this at all?

Reply:
Hi TheENGR,

I deleted the manifest.chrome.json file.  

Thanks.
Satya

Reply:
You need to push that deletion